app/helpers/goals_helper.rb

```diff
@@ -1,14 +1,18 @@
 module GoalsHelper
+  TIME_ALLOTTED_TO_COMPLETE_A_GOAL_IN_DAYS  = 7.days
```

Comment:
Good constants and well done putting the units in the name so its clear. 

Try running `7.days` in `rails c` and see what number you get in return. Review the name of `TIME_ALLOTTED_TO_COMPLETE_A_GOAL_IN_DAYS` afterward.